### PR TITLE
fix: my collection UI updates

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Fix**

This PR solves [CX-3065]

### Description
Checklist: 

1. Save changes - disable the button if no changes are made on the form
2. Demand Index - make padding 40
3. Titles in Insights - make title 20, subtitle (if exists) 13 (sm-display)
4. Demand Index - make number 40 (xl) and text 20 (medium)
5. Market insights - make the titles small-display
6. My collection Artwork - make “---” black60

| fix | before | after |
| --- | --- | --- |
| 1. Save changes button, disable when form is dirty | image | image |
| 2. Demand Index top padding | image | image |
| 3. Titles and subtitles in Insights | image | image |
| 4. Demand Index Section | image| image |
| 5. Market insights Section | image | image |
| 6. My collection Artwork details, empty value | image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3065]: https://artsyproduct.atlassian.net/browse/CX-3065?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionArtworkForm.jest.tsx [hunk 1]
```diff
@@ -187,10 +187,24 @@ describe("Edit artwork", () => {
...
+    it("enables save button if form is dirty", () => {
+      getWrapper().renderWithRelay({
+        Artwork: () => mockArtwork,
+      })
+      fireEvent.change(
+        screen.getByTestId("my-collection-artwork-details-title"),
+        {
+          target: { value: "Some new value" },
+        }
+      )
```
Comment (MrSltun) [inline src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionArtworkForm.jest.tsx:207]: 👏

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ const InsightColumn = ({ name, value }: { name: string; value: string }) => {
   return (
     <Column span={[6, 4, 2]}>
       <Flex flexDirection={"column"} justifyContent="flex-start">
-        <Text variant={["xs", "md", "md"]}>{name}</Text>
+        <Text variant={["xs", "sm-display", "sm-display"]}>{name}</Text>
         <Text variant={["lg", "xl", "xl"]}>{value}</Text>
```
Comment (MrSltun) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx:116]: **Suggestion:** 
```suggestion
        <Text variant={["xs", "sm-display"]}>{name}</Text>
        <Text variant={["lg", "xl"]}>{value}</Text>
```

**Review by MrSltun — APPROVED** [review]
Great work Daria!! ✨
I just left a small suggestion

File: src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionArtworkForm.jest.tsx (hunk 1 above)
Comment (dariakoko) [reply src/Apps/MyCollection/Routes/EditArtwork/__tests__/MyCollectionArtworkForm.jest.tsx:207]: 💃

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx]: ah oh cool, thanks, i thought there was a reason to keep all three

Comment (MrSltun) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx]: There is a reason to have them sometimes, here's a guide about them in [Palette] :)(https://palette.artsy.net/guides/responsive/)

Comment (dariakoko) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx]: Got it :)

**Pull request merged**